### PR TITLE
feature(gry): add acrobot env and dqn config

### PR DESCRIPTION
**Pull request by ruoyuGao**
## Description
add acrobot env into the classic control. And provide a config file for dqn
the result dqn achieved is here: $-62\pm1$ with  8 evaluate episodes and 2M training steps
result:
reward_acrobot

## Check List

- [x] merge the latest version source branch/repo, and resolve all the conflicts
- [x] pass style check
- [x] pass all the tests

### CODE REVIEW

File: dizoo/classic_control/acrobot/envs/acrobot_env.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        self._init_flag = False
+        self._replay_path = None
+        self._observation_space = gym.spaces.Box(
+            low=np.array([-1.0, -1.0, -1.0, -1.0, -12.57, -28.27]),
+            high=np.array([1.0, 1.0, 1.0, 1.0, 12.57, 28.27]),
+            shape=(6, ),
+            dtype=np.float32
+        )
+        self._action_space = gym.spaces.Discrete(3)
+        self._action_space.seed(0)  # default seed
```
Comment (PaParaZz1) [inline dizoo/classic_control/acrobot/envs/acrobot_env.py:27]: why set seed here

File: dizoo/classic_control/acrobot/envs/acrobot_env.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        if not self._init_flag:
+            self._env = gym.make('Acrobot-v1')
+            if self._replay_path is not None:
+                self._env = gym.wrappers.RecordVideo(
+                    self._env,
+                    video_folder=self._replay_path,
+                    episode_trigger=lambda episode_id: True,
+                    name_prefix='rl-video-{}'.format(id(self))
+                )
+            if hasattr(self._cfg, 'obs_plus_prev_action_reward') and self._cfg.obs_plus_prev_action_reward:
```
Comment (PaParaZz1) [inline dizoo/classic_control/acrobot/envs/acrobot_env.py:40]: you can remove this wrapper in this env

File: dizoo/classic_control/acrobot/envs/acrobot_env.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        return BaseEnvTimestep(obs, rew, done, info)
+
+    def enable_save_replay(self, replay_path: Optional[str] = None) -> None:
+        if replay_path is None:
+            replay_path = './video'
+        self._replay_path = replay_path
+
+    def random_action(self) -> np.ndarray:
+        random_action = self.action_space.sample()
+        random_action = to_ndarray([random_action], dtype=np.int64)
```
Comment (PaParaZz1) [inline dizoo/classic_control/acrobot/envs/acrobot_env.py:84]: why indicate dtype here

File: dizoo/classic_control/acrobot/envs/test_acrobot_env.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                print(timestep)
+                assert isinstance(timestep.obs, np.ndarray)
+                assert isinstance(timestep.done, bool)
+                assert timestep.obs.shape == (6, )
+                assert timestep.reward.shape == (1, )
+                assert timestep.reward >= env.reward_space.low
+                assert timestep.reward <= env.reward_space.high
+        print(env.observation_space, env.action_space, env.reward_space)
+        env.close()
+
```
Comment (PaParaZz1) [inline dizoo/classic_control/acrobot/envs/test_acrobot_env.py:35]: remove empty lines

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

**Comment by PaParaZz1** [comment]
add the information of this env in [env table](https://github.com/opendilab/DI-engine#environment-versatility)

File: dizoo/classic_control/acrobot/envs/acrobot_env.py (hunk 1 above)
Comment (ruoyuGao) [reply dizoo/classic_control/acrobot/envs/acrobot_env.py:27]: keep this line here for future update

File: dizoo/classic_control/acrobot/envs/acrobot_env.py (hunk 2 above)
Comment (ruoyuGao) [reply dizoo/classic_control/acrobot/envs/acrobot_env.py]: removed

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**